WikiDrips/UImageExtensions.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TitleImageView.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/6/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import UIKit
+
+extension UIImage {
+    
+    func image(withInitials: String?, in bounds: CGRect) -> UIImage {
```

Comment:
If you're not using `self` then maybe this should be a static method instead of an instance method.